front/app/modules/commercial/id_vienna_saml/components/ViennaSamlButton.tsx

```diff
@@ -11,6 +11,20 @@ import { TOnContinueFunction } from 'components/SignUpIn/AuthProviderButton';
 // typings
 import { TSignUpInFlow } from 'components/SignUpIn';
 
+// styling
+import styled from 'styled-components';
+import { fontSizes } from 'utils/styleUtils';
+
+const Container = styled.div`
+  display: flex;
+  gap: 2px;
```

Comment:
Do we have fixed values for margins & gaps or shall I just freestyle it?

Reply:
For now, we don't have standard values but ideally everything should be divisible by 4 in preparation of introducing a spacing system in our component library.

Reply:
Okay, I guess 2 / 4 = 0.5 is cool then, because it creates a rational fractional number.

Reply:
Why not just make it 4px? And 12px instead of 10px above? Or does it mess up the layout?